flexidag/src/consensusdb/consensus_ghostdag.rs

```diff
@@ -232,15 +227,6 @@ impl DbGhostdagStore {
         }
         self.access
             .write(BatchDbWriter::new(batch), hash, data.clone())?;
-        self.compact_access.write(
-            BatchDbWriter::new(batch),
-            hash,
-            CompactGhostdagData {
```

Comment:
We don't need CompactGhostdagData? If we can remove struct CompactGhostdagData

Reply:
Currently, we can remove some CF from rocksdb, such as DagHeader and CompactGhostDag. Actually they're not used or partially used in dag. But the db size seems not become smaller for an overnight test. lol. Need more investigation.